neural_modelling/src/neuron/plasticity/synapse_dynamics.h

```diff
@@ -12,6 +12,9 @@ bool synapse_dynamics_process_plastic_synapses(
     address_t plastic_region_address, address_t fixed_region_address,
     weight_t *ring_buffers, uint32_t time);
 
+void synapse_dynamics_process_neuromodulator_event(
```

Comment:
Never pass non 32-bit data types to functions in ARM code - it results in loads of unnecessary bit-masking operations